### PR TITLE
feature: service worker

### PR DESCRIPTION
**Pull request by Ogenbertrand**
@stephane-segning 
implement the service worker in the power pay app.

[Add: service worker in the power pay app](https://github.com/ADORSYS-GIS/e2e-banking-app/pull/109/commits/c7bb1940c24cb77e6dc8edb0fa8b10393bec2072)

### CODE REVIEW

**Comment by Ogenbertrand** [comment]
@stephane-segning 
The service worker is modified i used pwa plugins to implement it.
[Add vite pwa plugins](https://github.com/ADORSYS-GIS/e2e-banking-app/pull/109/commits/a462d5f8f3674ee8cddc4c4cad72b1ba41141893)

File: power-pay-frontend/dev-dist/registerSW.js
Comment (stephane-segning) [inline power-pay-frontend/dev-dist/registerSW.js:1]: This folder `dev-dist` should be removed from the vcs

File: power-pay-frontend/dev-dist/sw.js
Comment (stephane-segning) [inline power-pay-frontend/dev-dist/sw.js:1]: This folder `dev-dist` should be removed from the vcs

File: power-pay-frontend/dev-dist/workbox-b5f7729d.js
Comment (stephane-segning) [inline power-pay-frontend/dev-dist/workbox-b5f7729d.js:1]: This folder `dev-dist` should be removed from the vcs

File: power-pay-frontend/src/main.tsx
Comment (stephane-segning) [inline power-pay-frontend/src/main.tsx:1]: Cancel unuseful changes

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work @Ogenbertrand !

File: power-pay-frontend/src/main.tsx
Comment (Ogenbertrand) [reply power-pay-frontend/src/main.tsx:1]: okay sir

**Comment by Ogenbertrand** [comment]
@stephane-segning 
[Remove dev-dist folder and its contents from VCS](https://github.com/ADORSYS-GIS/e2e-banking-app/pull/109/commits/fec549b62d9f7fd1b79a43a3ddf4521a3bd03827)

**Review by stephane-segning — APPROVED** [review]
Good work @Ogenbertrand !

**Pull request merged**